src/include/storage/index/bwtree_index.h

```diff
@@ -44,34 +44,98 @@ class BwTreeIndex final : public Index {
   mutable common::SpinLatch transaction_context_latch_;  // latch used to protect transaction context
 
  public:
+  /**
+   * @return type of the index. Note that this is the physical type, not extracted from the underlying schema or other
+   * catalog metadata. This is mostly used for debugging purposes.
+   */
   IndexType Type() const final { return IndexType::BWTREE; }
 
+  /**
+   * Invoke garbage collection on the index. For some underlying index types this may be a no-op.
```

Comment:
Could you remove the "no-op" part? Or better, "This is NOT a no-op", or something like that.